cub/test/catch2_test_device_partition_flagged.cu

```diff
@@ -99,7 +103,11 @@ using types =
                  std::uint32_t,
 // WAR bug in vec type handling in NVCC 12.0 + GCC 11.4 + C++20
 #if !(_CCCL_CUDA_COMPILER(NVCC, ==, 12, 0) && _CCCL_COMPILER(GCC, ==, 11, 4) && _CCCL_STD_VER == 2020)
+#  if _CCCL_CTK_AT_LEAST(13, 0)
+                 ulonglong4_16a,
+#  else // _CCCL_CTK_AT_LEAST(13, 0)
                  ulonglong4,
+#  endif // _CCCL_CTK_AT_LEAST(13, 0)
```

Comment:
I am wondering whether a type alias would have been better, but maybe as a follow-up, so we can merge this now

Reply:
Thanks a lot, lets do this in a followup